src/js/page/ui/toasts.js

```diff
@@ -1,17 +1,21 @@
 import { strToEl, transitionToClass } from '../utils.js';
 
 class Toast {
-  constructor(message, duration, buttons) {
+  constructor(message, duration, buttons, isError) {
```

Comment:
Move `isError` to an option to avoid the boolean trap?

Reply:
Not sure I follow you here... BTW we don't call `Toast` anywhere else except from the `Toasts` Class.

Reply:
At some point I played with moving the Toast class to a separate file, but I went back and forth because it's only used here. If you think that would be better let me know.